packages/webapp/src/components/ui/FormGenerator/FormFieldManager.ts

```diff
@@ -71,14 +87,14 @@ export class FormFieldManager {
 		for (const field of this.fields) {
 			if (isRequired(field) && !this.isFieldValueRecorded(field)) {
 				log(`validation error: field ${field.name} is required and not present`, this.value)
-				this.addFieldError(field.id, t('formGenerator.validation.required'))
+				this.addFieldError(field.id, t('formGenerator.validation.requiredField'))
```

Comment:
why change the key? Is the existing key still in use or should it be removed as well from the language files?